app/components/molecules/PuzzlePrint/PuzzlePrintHeader.tsx

```diff
@@ -0,0 +1,34 @@
+import { Heading, Stack, HStack, Button, Text, Icon } from '@chakra-ui/react'
+
+import { Printer, ArrowLeft } from 'lucide-react'
+
+import Link from '~/components/atoms/Link'
+
+const PuzzlePrintHeader = () => (
+  <Stack
+    direction={{
+      base: 'column',
+      md: 'row',
+    }}
+    width="full"
+    justifyContent="space-between"
+    sx={{
+      '@media print': {
+        display: 'none',
+      },
+    }}
+  >
+    <Heading>Exportation des QRCodes</Heading>
+    <HStack>
+      <Button leftIcon={<Icon as={Printer} />} onClick={() => window.print()}>
+        Imprimer
+      </Button>
+      <Button as={Link} to="/admin/puzzles" gap={1}>
+        <Icon as={ArrowLeft} />
+        <Text>Retour</Text>
```

Comment:
Un `p`dans un `button` n'est pas valide
```suggestion
        <Text as="span">Retour</Text>
```